Makefile

```diff
@@ -263,6 +269,13 @@ endif
 	rm -f $(DESTDIR)$(bindir)/add.modules
 	rm -f $(DESTDIR)$(bindir)/modulecmd
 	rm -f $(DESTDIR)$(bindir)/mkroot
+ifeq ($(vimplugin),y)
+	rm -f $(DESTDIR)$(vimplugindir)/ftdetect/modulefile.vim
+	rm -f $(DESTDIR)$(vimplugindir)/ftplugin/modulefile.vim
+	rm -f $(DESTDIR)$(vimplugindir)/syntax/modulefile.vim
+	-rmdir $(DESTDIR)$(vimplugindir)/{ftdetect,ftplugin,syntax}
```

Comment:
the braces `{}` are not understood by make (you can see [the make uninstall failing on the CI](https://travis-ci.org/cea-hpc/modules/builds/466635254?utm_source=github_status&utm_medium=notification))

Reply:
Mmh, interesting - I guess it's more a shell thing … I tested with `zsh` / `tcsh`  and both supported it, so it worked fine… maybe `sh` does not…

Reply:
looks like the other problems… anyway replaced it